_topic_maps/_topic_map_osd.yml

```diff
@@ -249,6 +249,81 @@ Topics:
     - Name: Configuring custom domains for applications
       File: osd-config-custom-domains-applications
 ---
+Name: Logging
+Dir: logging
+Distros: openshift-dedicated
+Topics:
+- Name: Release notes
+  File: cluster-logging-release-notes
+- Name: About Logging
+  File: cluster-logging
+  Name: Accessing the service logs
```

Comment:
@eohartman hi! Can you please prepend this line with a hyphen? Please also resolve the merge conflict and then check if a subsequent push triggers the Travis check. I made these two changes only in https://github.com/openshift/openshift-docs/pull/52821 and the checks ran. Note that the checks in my test produced failure results on account of cross reference issues, but the checks ran, at least.

Please let me know if those suggested changes start the checks in your PR. Thanks!